### ISSUE TITLE
ide-helper:models Makes no effect

### ISSUE THREAD

**Issue by mityukov** [barryvdh/laravel-ide-helper#978]
Laravel version: 5.8.38
PHP 7.3.15

When I try to use this command, I have no annotations added to the class. If I choose not to write to the class — it creates empty _ide_helper_models.php:
```
<?php

// @formatter:off
/**
 * A helper file for your Eloquent Models
 * Copy the phpDocs from this file to the correct Model,
 * And remove them from this file, to prevent double declarations.
 *
 * @author Barry vd. Heuvel <barryvdh@gmail.com>
 */

```

What's the worst: No errors/warnings/ in logs. No messages of any kind even with -vvv tag.

Models location seems to be correct, in the config file:
```
    // config/ide-helper.php

    'model_locations' => array(
        'app',
    ),
```

This is what I see in the console:
![image](https://user-images.githubusercontent.com/1751910/85895538-f6049a00-b807-11ea-8ece-a798aed2cb7f.png)

**Comment by mfn** [comment]
Which version of ide-helper? :)

**Comment by mfn** [comment]
Is the database connection working correctly? This is required for this command

There are _some_ exception catch parts which swallow it 😬, maybe one of them was triggered. Can you add some debug statement to them?

**Comment by mityukov** [comment]
IDE Helper version : * v2.7.0

> Is the database connection working correctly? This is required for this command

DB connection is alright in tinker and/or artisan commands. Not sure how to check it specifically for ide-helper.

**Comment by mfn** [comment]
Where is your model(s) exactly located?

Please paste the code of a model which does not work (remove/replace sensitive information).

**Comment by nuke99** [comment]
Same. it doesn't work when I do `php artisan ide-helper:models User `

**Comment by mityukov** [comment]
@mfn 

Models are all located in app/ directory. One model example:
```php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use App\Traits\HasSubdomainScope;

class CustomField extends Model
{
    use HasSubdomainScope;

    public $timestamps = false;

    protected $guarded = [];

    public function scopeTypeLead($query)
    {
        return $query->where('element_type', 'lead');
    }
    public function scopeTypeContact($query)
    {
        return $query->where('element_type', 'contact');
    }
    public function scopeTypeCompany($query)
    {
        return $query->where('element_type', 'company');
    }

    public function getHtmlAttribute()
    {
        switch ($this->field_type) {
            case \Amo\CustomField::CF_TYPE_BIRTHDAY:
            case \Amo\CustomField::CF_TYPE_DATE:
                return date('d.m.y', strtotime($this->value));
            case \Amo\CustomField::CF_TYPE_CHECKBOX:
                return $this->value ? 'Да' : 'Нет';
            case \Amo\CustomField::CF_TYPE_MULTISELECT:
                return $this->subdomain($this->subdomain)
                    ->where('element_type', $this->element_type)
                    ->where('element_id', $this->element_id)
                    ->where('field_id', $this->field_id)
                    ->pluck('value')->implode(', ');
            default:
                return $this->value;
        }
    }

    public function getFieldTypeAttribute()
    {
        if ($this->element_type === 'contact') {
            return (int)amo($this->subdomain)->getContactCustomFieldById($this->field_id)['type_id'];
        }

        return (int)amo($this->subdomain)->getLeadCustomFieldById($this->field_id)['type_id'];
    }
}

```

**Comment by mfn** [comment]
Nothing stands out to me; also since you said _all_ models are affected, i.e basically none works…

Two suggestions:
- put up a public repo on github showing the problem
- start debugging yourself
  I would probably start looking what `\Barryvdh\LaravelIdeHelper\Console\ModelsCommand::loadModels` "finds" and take it from there.
  - does it discover the models?
  - in the main `foreach` loop starting at https://github.com/barryvdh/laravel-ide-helper/blob/f3d841c0a30645a8bbf908e89ab31acf0053ad93/src/Console/ModelsCommand.php#L209 there are some conditions skipping the model; is any of them triggered?

**Comment by ollycross** [comment]
I was having a similar issue to this.  Running `php artisan ide-helper:models` generated the correct `_ide_helper_models.php` file with content, but running `php artisan ide-helper:models \App\Development` (where `Development` is my Model) resulted in an empty file, as described above.

Turns out I was being an idiot and I needed to run `'php artisan ide-helper:models '\App\Development'` (note the single quotes).  Presumably the namespace separators are being interpreted by the shell as escape characters and causing the model not to be found.

Anyway, with the quotes it works.
